client/src/pages/ViewRequestStatus/DonationRequestStatus.js

```diff
@@ -0,0 +1,27 @@
+
+import React, { useEffect, useState } from "react";
+import './StatusView.css'
+import { useParams } from "react-router-dom";
+
+
+function DonationRequestStatues() {
+    const [request, setRequest] = useState(null);
+	// this helps get the id from the router
+	const { id } = useParams();
+
+    //  this use effect/ it may be not the right Api that we are going to Fetch the data with
+    // this only template 
+	useEffect(() => {
+		fetch(`/laptop_assignment/{id}`)
```

Comment:
it should be laptop_donation. But for now, this is not part of the ticket at all. You don't need the useEffect